File: src/main/resources/static/js/service/chatService.js
//服务层
app.service('chatService',function($http){
	    	
	//读取列表数据绑定到表单中
	this.findAll=function(){
		return $http.get('../chat/findAll.do');		
	}
	//分页 
	this.findPage=function(page,rows){
		return $http.get('../chat/findPage.do?page='+page+'&rows='+rows);
	}
	//查询实体
	this.findOne=function(id){
		return $http.get('../chat/findOne.do?id='+id);
	}
	//增加 
	this.add=function(entity){
		return  $http.post('../chat/add.do',entity );
	}
	//修改 
	this.update=function(entity){
		return  $http.post('../chat/update.do',entity );
	}
	//删除
	this.dele=function(ids){
		return $http.get('../chat/delete.do?ids='+ids);
	}
	//搜索
	this.search=function(page,rows,searchEntity){
		return $http.post('../chat/search.do?page='+page+"&rows="+rows, searchEntity);
	}    	
});

File: src/main/java/com/zzti/fruits/service/ChatService.java
package com.zzti.fruits.service;
import java.util.List;

import com.zzti.fruits.entity.PageResult;
import com.zzti.fruits.pojo.Chat;

/**
 * 服务层接口
 * @author Administrator
 *
 */
public interface ChatService {

	/**
	 * 返回全部列表
	 * @return
	 */
	public List<Chat> findAll();
	
	
	/**
	 * 返回分页列表
	 * @return
	 */
	public PageResult findPage(int pageNum, int pageSize);
	
	
	/**
	 * 增加
	*/
	public void add(Chat chat);
	
	
	/**
	 * 修改
	 */
	public void update(Chat chat);
	

	/**
	 * 根据ID获取实体
	 * @param id
	 * @return
	 */
	public Chat findOne(Long id);
	
	
	/**
	 * 批量删除
	 * @param ids
	 */
	public void delete(Long[] ids);

	/**
	 * 分页
	 * @param pageNum 当前页 码
	 * @param pageSize 每页记录数
	 * @return
	 */
	public PageResult findPage(Chat chat, int pageNum, int pageSize);
	
}
